### PR TITLE
Lesson3

### PR DESCRIPTION
**Pull request by saumya665**

### CODE REVIEW

File: printNamesExpress.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//program to print with express
+const fs = require("fs");
+const express = require("express");
+const app = express();
+app.listen(3615);
+const myfunc = (re1,re2)=>{
```
Comment (allopez7) [inline printNamesExpress.js]: I don't know what re1, re2 are. If someone new was reading these two, they would have no idea what they are. Just use the req, res. It makes it easier to understand.

**Review by allopez7 — CHANGES_REQUESTED** [review]
Simple change.

**Review by allopez7 — APPROVED** [review]
Good Job!

File: printNamesExpress.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const express = require("express");
+const app = express();
+app.listen(3615);
+const myfunc = (req,res)=>{
+  const printFunc=(err,files)=>{
+    let allNames = " ";
+    files.forEach((name)=>{
+      if(name!="saumya")
+        allNames = allNames + " " +name;
+  })
```
Comment (chocobrownie) [inline printNamesExpress.js]: semicolomn after you finish a line, the indentation is a bit wierd

File: printNamesExpress.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    res.send(allNames);
+  };
+  fs.readdir("/home", printFunc);
+};
+app.get("/name", myfunc);
```
Comment (chocobrownie) [inline printNamesExpress.js]: what does myfunc mean? try to specify

File: printNamesExpress.js (hunk 2 above)
Comment (saumya665) [reply printNamesExpress.js]: fixed the semicolon. don't know how to fix indentation

Comment (saumya665) [reply printNamesExpress.js]: to fix indentation do gg then =G

**Comment by seemcat** [comment]
Approve!

**Review by seemcat — APPROVED** [review]

**Pull request merged**